index.json

```diff
@@ -11269,14 +11269,14 @@
           "location": "body"
         },
         {
-          "name": "dismissal_restrictions.users",
+          "name": "required_pull_request_reviews.dismissal_restrictions.users",
           "type": "string[]",
           "description": "The list of user `login`s with dismissal access",
           "required": false,
           "location": "body"
         },
         {
-          "name": "dismissal_restrictions.teams",
+          "name": "required_pull_request_reviews.dismissal_restrictions.teams",
```

Comment:
my fault, but this is a regression, `dismissal_restrictions.*` is correct in this place, see
https://developer.github.com/v3/repos/branches/#update-pull-request-review-enforcement-of-protected-branch

I’ll fix that

Reply:
Didn't get it. :confused: